File: Web.Admin/ClientApp/src/app/models/Organization/Organization.ts
import { Address } from '../Address/Address';
import { PersonLite } from '../Person/PersonLite';

export class Organization {

  id: number;
  name: string;
  url: string;
  addresses: Address[];
  voiceNumber: string;
  faxNumber: string;
  parent: Organization;
  persons: PersonLite[];
  emails: string[];

  public constructor(id: number,
    name: string,
    url: string,
    addresses: Address[],
    voiceNumber: string,
    faxNumber: string,
    parent: Organization) {
    this.id = id;
    this.name = name;
    this.url = url;
    this.addresses = addresses;
    this.voiceNumber = voiceNumber;
    this.faxNumber = faxNumber;
    this.parent = parent;
  };

}

File: Libraries/Shared/NIST.CVP.Libraries.Shared.ACVPCore.Abstractions/Models/Organization.cs
using System;
using System.Collections.Generic;

namespace NIST.CVP.Libraries.Shared.ACVPCore.Abstractions.Models
{
    public class Organization
    {
        public long ID { get; set; }
        public string Name { get; set; }
        public string Url { get; set; }
        public string VoiceNumber { get; set; }
        public string FaxNumber { get; set; }
        public OrganizationLite Parent { get; set; }
        public List<Address> Addresses { get; set; }
        public List<PersonLite> Persons { get; set; }
        public List<String> Emails { get; set; }
    }
}
